### ISSUE TITLE
Error in loading model from HARK

### ISSUE THREAD

**Issue by mbholm** [econ-ark/HARK#216]
I tried loading a model from HARK, but I get the following error: 

```
from HARK.ConsumptionSaving.ConsIndShockModel import IndShockConsumerType
Traceback (most recent call last):

  File "<ipython-input-1-3276f49379bc>", line 1, in <module>
    from HARK.ConsumptionSaving.ConsIndShockModel import IndShockConsumerType

  File "C:\Users\mbm\AppData\Local\Continuum\anaconda2\lib\site-packages\HARK\ConsumptionSaving\ConsIndShockModel.py", line 24, in <module>
    from HARK import AgentType, Solution, NullFunc, HARKobject

ImportError: cannot import name AgentType
```

Simply changing the line "from HARK import AgentType, Solution, NullFunc, HARKobject" to "from HARK.core import AgentType, Solution, NullFunc, HARKobject" solved the issue, but this should then be changed on the package distribution on pip.

**Comment by mnwhite** [comment]
Are you using the latest version of HARK? Several people have reported this
issue, but it's a version issue. Try:

`pip install econ-ark --update`

**Comment by mbholm** [comment]
It seems I wasn’t. I thought I updated the whole thing when I wrote ‘pip install HARK --upgrade’ but that was wrong (it was supposed to be econ-ark). I have updated the package now and it works fine. Sorry for the unnecessary trouble.

Martin


Fra: Matthew N. White <notifications@github.com>
Sendt: mandag 14. januar 2019 15:33
Til: econ-ark/HARK <HARK@noreply.github.com>
Kopi: Martin Blomhoff Holm <martin.b.holm@outlook.com>; Author <author@noreply.github.com>
Emne: Re: [econ-ark/HARK] Error in loading model from HARK (#216)

Are you using the latest version of HARK? Several people have reported this
issue, but it's a version issue. Try:

`pip install econ-ark --update`

On Mon, Jan 14, 2019, 7:43 AM Martin Blomhoff Holm <notifications@github.com
wrote:

<mailto:notifications@github.com%0bwrote:%0b%0b>> I tried loading a model from HARK, but I get the following error:
